dsp-repository/v1/dsp-repository.shacl.ttl

```diff
@@ -26,32 +26,670 @@
+dsp-repo:hasDateCreated
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasDateModified
+dsp-repo:hasDateModified
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasDistribution
+dsp-repo:hasDistribution
+    a rdf:Property ;
+    rdfs:subPropertyOf schema:distribution .
+
+# http://ns.dasch.swiss/repository#hasGivenName
+dsp-repo:hasGivenName
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasFamilyName
+dsp-repo:hasFamilyName
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasEmail
+dsp-repo:hasEmail
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasAddress
+dsp-repo:hasAddress
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#isMemberOf
+dsp-repo:isMemberOf
+    a rdf:Property ;
+    rdfs:subPropertyOf schema:memberOf .
+
+# http://ns.dasch.swiss/repository#hasJobTitle
+dsp-repo:hasJobTitle
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasRole
+dsp-repo:hasRole
+    a rdf:Property ;
+    rdfs:subPropertyOf prov:hadRole .
+
+# http://ns.dasch.swiss/repository#hasNumber
+dsp-repo:hasNumber
+    a rdf:Property .
+
+# # http://ns.dasch.swiss/repository#hasBooBoo
+# dsp-repo:hasBooBoo
+#     a rdf:Property .
+
+##### CLASS DEFINITIONS #####
+### --- Class definition of dsp-repo:Project --- 
+# http://ns.dasch.swiss/repository#Project
+dsp-repo:Project 
+    a rdfs:Class ;
+    rdfs:label "dsp-repo:Project"@en ;
+    rdfs:comment "A project"@en ;
+    rdfs:subClassOf schema:Project , prov:Entity .   
+
+### --- Class definition of dsp-repo:Dataset --- 
+# http://ns.dasch.swiss/repository#Dataset
+dsp-repo:Dataset 
+    a rdfs:Class ;
+    rdfs:comment "A dataset"@en ;
+    rdfs:label "dsp-repo:Dataset"@en ;
+    rdfs:subClassOf schema:Dataset .
+
+### --- Class definition of dsp-repo:Person --- 
+# http://ns.dasch.swiss/repository#Person
+dsp-repo:Person 
+    a rdfs:Class ;
+    rdfs:comment "A person"@en ;
+    rdfs:label "dsp-repo:Person"@en ;
+    rdfs:subClassOf schema:Person , prov:Agent . 
+
+### --- Class definition of dsp-repo:Organization --- 
+# http://ns.dasch.swiss/repository#Organization
+dsp-repo:Organization 
+    a rdfs:Class ;
+    rdfs:comment "An organization"@en ;
+    rdfs:label "dsp-repo:Organization"@en ;
+    rdfs:subClassOf schema:Organization , prov:Agent . 
+
+### --- Class definition of dsp-repo:Grant --- 
+# http://ns.dasch.swiss/repository#Grant
+dsp-repo:Grant 
+    a rdfs:Class ;
+    rdfs:comment "A grant"@en ;
+    rdfs:label "dsp-repo:Grant"@en ;
+    rdfs:subClassOf schema:Grant . 
+
+### --- Class definition of dsp-repo:DataManagementPlan --- 
 # http://ns.dasch.swiss/repository#DataManagementPlan
 dsp-repo:DataManagementPlan
-    a owl:Class, sh:NodeShape ;
-    rdfs:label "dsp-repo:DataManagementPlan"@en ;
+    a rdfs:Class ;
     rdfs:comment "A Data Management Plan"@en ;
-    rdfs:subClassOf schema:Thing ;
+    rdfs:label "dsp-repo:DataManagementPlan"@en ;
+    rdfs:subClassOf schema:Thing .
```

Comment:
I would rather use `rdfs:subClassOf schema:CreativeWork` than `rdfs:subClassOf schema:Thing` which doesn't say much anything about the class.

Reply:
I would leave it to `schema:Thing` because everything is a thing, even `schema:CreativeWork`, please see: https://schema.org/Thing

Reply:
changed to ` rdfs:subClassOf schema:CreativeWork`